### PR TITLE
DOCSP-15896 update cursor.map return type

### PR DESCRIPTION
**Pull request by davemungo**

### CODE REVIEW

**Review by addaleax — COMMENTED** [review]
Looks good, but I think it’s important to note that it’s the same cursor, i.e. `.map()` changes the existing cursor rather than creating a new one.

**Comment by davemungo** [comment]
@addaleax I've added a note about the cursor. Would you have a look?

**Review by addaleax — APPROVED** [review]

File: source/release-notes/5.0-compatibility.txt [hunk 1]
```diff
@@ -188,6 +188,14 @@ You can opt out of this behavior by setting the cluster-wide
 :ref:`read concern <read-concern>` with 
 :dbcommand:`setDefaultRWConcern`.
 
+New ``cursor.map()`` Return Type
+--------------------------------------
```
Comment (jeff-allen-mongo) [inline source/release-notes/5.0-compatibility.txt:192]: This underline needs to be shortened to match the text length.

**Comment by jeff-allen-mongo** [comment]
Merged in https://github.com/mongodb/docs/pull/5662

**Pull request closed without merging**